keras_nlp/benchmarks/text_generation_config.py

```diff
@@ -0,0 +1,66 @@
+# Copyright 2022 The KerasNLP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+COMMON_ARGS = {
```

Comment:
Should we just move these straight into the main file? There's a lot of hardcoding one from the other. If we moved these in, we could get rid of `SUPPORTED_TEXT_GEN_METHODS`, `EXECUTION_METHODS` and the validation we are doing there.

I also think for people that want to run the benchmarks with modifications, having a single file to edit might be nice.